robottelo/ui/hosts.py

```diff
@@ -194,6 +195,12 @@ def update_host_bulkactions(
                         parameter['command']
                     )
                     self.click(common_locators['submit'])
+                if action == 'Delete Hosts':
+                    self.click(locators['host.bulk_submit'])
+                    return self.wait_until_element(
+                        locators['task.finished_state'],
+                        timeout=parameter['timeout'],
```

Comment:
by the other side if there is no parameter list the action will not execute, so let it as is (is the best option)